playbook/app/pb_kits/playbook/pb_phone_number_input/_phone_number_input.scss

```diff
@@ -0,0 +1,90 @@
+@import "../tokens/colors";
+
+.pb_phone_number_input {
+  input::placeholder {
+    color: $focus_input_light;
+  }
+
+  .iti__country.iti__highlight {
+    background-color: $hover_light;
+
+    .iti__country-name, .iti__dial-code {
+      color: $primary
+    }
+  }
+
+  .iti--allow-dropdown .iti__flag-container:hover .iti__selected-flag {
+    background-color: $focus_input_light;
+  }
+
+  .iti__flag-container:hover + .text_input {
+    background-color: $focus_input_light;
+  }
+
+  .iti__flag {
+    background-image: url("https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.19/img/flags.png");
```

Comment:
Can we not host this ourselves?

Reply:
Sure! But we would not be subject to updates from the library. And we would need to manually update the png ourselves if and when new countries are added.

Reply:
Is there not an npm package? If this link goes down we will be unable to use the kit

Reply:
@jasperfurniss and I paired to try to use the flags png file from the npm package using the background image property, but we are having problems with the MiniCssExtractPlugin. Since the file is too big, we decided not to bring the file into the project.
We will create a task to handle this separately due to the complexity.